src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+
+* If import-only mode was not used, the source image is automatically deleted.
+* The target image becomes fully independent.
+
+**Syntax:**
+
+```
+rbd migration commit TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration commit target_pool1/target_image1
+```
+
+#### Option 2: Abort the Migration
+
+Migrations can be cancelled.  Cancelling a migration will cause the following to happen:
+
+* Remove any cross-links between the images.
+* Delete the target image, restoring the source image to its previous state.
+
+**Syntax:**
+
+```
+rbd migration abort TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration abort targetpool1/targetimage1
+```
+
+The following example shows how to migrate data from one Ceph cluster to another Ceph
+cluster, here named _cluster1_ and _cluster2:_
+
+```
+[ceph: root@rbd-client /]# cat /tmp/native_spec
+{
+  "cluster_name": "c1",
+  "type": "native",
+  "pool_name": "pool1",
+  "image_name": "image1",
+  "snap_name": "snap1"
+}
+[ceph: root@rbd-client /]# rbd migration prepare --import-only --source-spec-path /tmp/native_spec c2pool1/c2image1 --cluster c2
+[ceph: root@rbd-client /]# rbd migration execute c2pool1/c2image1 --cluster c2
+Image migration: 100% complete...done.
+[ceph: root@rbd-client /]# rbd migration commit c2pool1/c2image1 --cluster c2
+Commit image migration: 100% complete...done.
+```
+
+### Supported Image Formats
+
+Live migration supports three primary formats:
+
+1. *Native format* uses Ceph's internal format for efficient migration.
+
+The native format does not include the stream since it utilizes
+native Ceph operations. For example, to import from the
+image ``rbd/ns1/image1@snap1``, the source specification can be constructed as below:
+
+```
+{
+"type": "native",
+"pool_name": "rbd",
+"pool_namespace": "ns1",
+"image_name": "image1",
+"snap_name": "snap1"
+}
+```
+
+2. *QCOW format* is compatible with QEMU Copy-On-Write volume images.
+
+The QCOW format describes a QEMU copy-on-write (QCOW) block device.
+Both QCOW v1 and v2 formats are currently supported, with the exception of
+certain features including compression, encryption, backing files, and external
+data files. Use the QCOW format with any supported stream source:
+
+```
+{
+    "type": "qcow",
+    "stream": {
+      "type": "file",
+  "file_path": "/mnt/image.qcow"
+    }
+}
+```
+
+3. *Raw format* is used for thick-provisioned block device exports.
+
+```
+{
+    "type": "raw",
+    "stream": {
+      "type": "file",
+      "file_path": "/mnt/image-head.raw"
+    },
+    "snapshots": [
+        {
+            "type": "raw",
+            "name": "snap1",
+            "stream": {
+              "type": "file",
+       "file_path": "/mnt/image-snap1.raw"
+            }
+        },
+    (optional oldest to newest ordering of snapshots)
+}
+```
+
+### Supported Streams
+
+Live migration supports multiple stream types for importing external data sources:
```

Comment:
This sentence is confusing because of the multivalence of "supports".

Maybe "Multiple stream types can be used to import external data sources during live migration."